.github/workflows/ci.yml

```diff
@@ -35,21 +35,85 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: "Build Package"
-        uses: pyansys/pydpf-actions/build_package@v2.2
+        uses: pyansys/pydpf-actions/build_package@v2.2.dev1
         with:
           python-version: ${{ matrix.python-version }}
           ANSYS_VERSION: ${{env.ANSYS_VERSION}}
           PACKAGE_NAME: ${{env.PACKAGE_NAME}}
           MODULE: ${{env.MODULE}}
           dpf-standalone-TOKEN: ${{secrets.DPF_PIPELINE}}
           install_extras: plotting
-          wheelhouse: "false"
+          wheelhouse: false
+          extra-pip-args: "--pre --find-links .github/workflows/"
 
-      - name: "Test Package"
-        uses: pyansys/pydpf-actions/test_package@v2.2.dev0
+      - name: "Prepare Testing Environment"
+        uses: pyansys/pydpf-actions/prepare_tests@v2.2
+        with:
+          DEBUG: true
+
+      - name: "Test Docstrings"
+        uses: pyansys/pydpf-actions/test_docstrings@v2.2
         with:
           MODULE: ${{env.MODULE}}
-          DEBUG: True
+          PACKAGE_NAME: ${{env.PACKAGE_NAME}}
+
+      - name: "Separate long Core tests"
+        shell: pwsh
+        run: |
+          New-Item -Path ".\" -Name "local_server_test" -ItemType "directory"
```

Comment:
@PProfizi @cbellot000 just that you know, pytest as an --ignore flag that can be used as:
pytest ./tests --ignore tests\test_server.py --ignore tests\test_launcher.py